bot/cogs/filtering.py

```diff
@@ -437,7 +506,7 @@ async def _has_rich_embed(msg: Message) -> bool:
                     if not embed.url or embed.url not in urls:
                         # If `embed.url` does not exist or if `embed.url` is not part of the content
                         # of the message, it's unlikely to be an auto-generated embed by Discord.
-                        return True
+                        return msg.embeds
```

Comment:
This change breaks the following check:

```py
if name == "filter_invites" and match is not True:
```

It will also break the following lines since trying to use `.items()` will fail on a list.

```py
for invite, data in match.items():
```

Reply:
That change applies to the `watch_rich_embeds` filter so the first check will always fail since it is not the `filter_invites` check, which remains the same